test/NCalc.Tests/Fixtures.cs

```diff
@@ -211,32 +212,32 @@ public void ShouldDisplayErrorMessages()
         [Fact]
         public void Maths()
         {
-            Assert.Equal(1M, new Expression("Abs(-1)").Evaluate());
-            Assert.Equal(0d, new Expression("Acos(1)").Evaluate());
-            Assert.Equal(0d, new Expression("Asin(0)").Evaluate());
-            Assert.Equal(0d, new Expression("Atan(0)").Evaluate());
-            Assert.Equal(2d, new Expression("Ceiling(1.5)").Evaluate());
-            Assert.Equal(1d, new Expression("Cos(0)").Evaluate());
-            Assert.Equal(1d, new Expression("Exp(0)").Evaluate());
-            Assert.Equal(1d, new Expression("Floor(1.5)").Evaluate());
-            Assert.Equal(-1d, new Expression("IEEERemainder(3,2)").Evaluate());
-            Assert.Equal(0d, new Expression("Log(1,10)").Evaluate());
-            Assert.Equal(0d, new Expression("Log10(1)").Evaluate());
-            Assert.Equal(9d, new Expression("Pow(3,2)").Evaluate());
-            Assert.Equal(3.22d, new Expression("Round(3.222,2)").Evaluate());
-            Assert.Equal(-1, new Expression("Sign(-10)").Evaluate());
-            Assert.Equal(0d, new Expression("Sin(0)").Evaluate());
-            Assert.Equal(2d, new Expression("Sqrt(4)").Evaluate());
-            Assert.Equal(0d, new Expression("Tan(0)").Evaluate());
-            Assert.Equal(1d, new Expression("Truncate(1.7)").Evaluate());
+            Assert.Equal(1M, new Expression("Abs(-1)", CultureInfo.InvariantCulture).Evaluate());

```

Comment:
The tests got a bit cluttered with CultureInfo.InvariantCulture added.  Perhaps a static helper in the test code might limit some of the noise.

```
        private static Expression CreateExpression(string expression, CultureInfo? cultureInfo = null) => 
            new Expression(expression, cultureInfo ?? CultureInfo.InvariantCulture);
```

Reply:
Sounds good, will make the changes tomorrow (I'm on +3 UTC)